storagenode/storagenodedb/database.go

```diff
@@ -87,26 +143,76 @@ func NewTest(log *zap.Logger, storageDir string) (*DB, error) {
 	}
 	pieces := filestore.New(piecesDir)
 
-	infodb, err := NewInfoTest()
-	if err != nil {
-		return nil, err
-	}
-
-	return &DB{
+	db := &DB{
 		log: log,
 
 		pieces: pieces,
-		info:   infodb,
 
 		kdb: teststore.New(),
 		ndb: teststore.New(),
 		adb: teststore.New(),
-	}, nil
+	}
+
+	versionsPath := ""
+	versionsDb, err := openTestDatabase()
+	if err != nil {
+		return nil, err
+	}
+
+	// Initialize databases. Currently shares one info.db database file but
+	// in the future these will initialize their own database connections.
+	db.versions = newVersions(versionsDb, versionsPath)
```

Comment:
why not make all these part  of the `db := &DB{...` above?

Reply:
done!